### PR TITLE
INTERNAL | Update data_policy to 2.0.0 for Drupal 9.2 compatibility

### PR DESCRIPTION
**Pull request by tbsiqueira**
## Problem
Update data policy module to use the new beta version

## How to test
- [ ] Test the data policy module

## Screenshots
N/A

## Release notes
N/A

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2657-mf6b5b6hoypivnzjen9kzjsdrtobt8hf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61bb069e5f46ed13d80feefb

**Review by navneet0693 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -92,9 +92,6 @@
                 "Undefined property: stdClass::$region in field_group_form_process().": "https://www.drupal.org/files/issues/2020-06-15/3059614-37.patch",
                 "Remove dependency on jQuery UI Accordion for the accordion field group formatter": "https://www.drupal.org/files/issues/2021-10-16/3154304-2.patch"
             },
-            "drupal/data_policy": {
-                "9.2.x compatibility": "https://git.drupalcode.org/project/data_policy/-/merge_requests/12.diff"
```
Comment (Ressinel) [inline composer.json:96]: @tbsiqueira I would recommend adding the patch as the file from the issue instead of MR.

Comment (navneet0693) [reply composer.json:96]: @Ressinel This change is already available in the new version.

Comment (Ressinel) [reply composer.json:96]: @navneet0693 What new version do you mean?

Comment (tbsiqueira) [reply composer.json:96]: @Ressinel, we released a 2.0.0-beta1 version for data_policy containing the latest needed changes, this is why we removed the patch and changes the module version on composer.json

Comment (Ressinel) [reply composer.json:96]: @tbsiqueira  check, thanks!

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2657-mf6b5b6hoypivnzjen9kzjsdrtobt8hf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61bb069e5f46ed13d80feefb

**Pull request merged**